### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/core/howto/plugin.xhtml`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4875]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4875|
|Type|enhancement|
|Created|2011-02-11 05:41:06Z|
|Branch|https://github.com/twisted/twisted/tree/plugin-sphinx-4875|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the plugin howtos

<details><summary>Searchable metadata</summary>

```
trac-id__4875 4875
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_plugin_sphinx_4875 branches/plugin-sphinx-4875
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297402866000000 1297402866000000
changetime__1300068906000000 1300068906000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30682]) Branching to 'plugin-sphinx-4875'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30683]) Merge the plugin parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs #4568
refs [#4875](https://github.com/twisted/twisted/issues/4875)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_6"></a>|@lvh set status to `closed`|
|-|-|

(In [30994]) Merge plugin.xhtml changes from plugin-sphinx-4875.

Author: screwtape, jdb
Reviewer: lvh
Fixes: [#4875](https://github.com/twisted/twisted/issues/4875)
  
Fixes formatting changes needed to make sphinx happy with plugin.xhtml.
